torch_xla/_internal/tpu.py

```diff
@@ -343,3 +343,9 @@ def configure_multiprocess(self, local_rank, local_world_size):
 
   def physical_chip_count(self):
     return num_available_chips()
+
+  def client_create_options(self):
+    return {
+        'max_inflight_computations':
+            xu.getenv_as('XLA_TPU_MAX_INFLIGHT_COMPUTATIONS', int, 4)
```

Comment:
shall we track this new variable in https://github.com/pytorch/xla/blob/master/configuration.yaml?